docs/components/secnav/index.html

```diff
@@ -0,0 +1,138 @@
+---
+title: Sec Nav
+minver: 1.1.0
+also:
+  components/secnav/test.html: Full Page Demo - Secondary Navigation
+  elements/hx-navigation-control: <hx-navigation-control>
+---
+  {% extends 'component.njk' %}
+
+  {% block page_header %}
+  <p>
+    The {{page.title}} is the secondary navigation menu bar across all breakpoints within the application frame of a HelixUI interface.
+    Use the {{page.title}} to provide access to globally available features.
+  </p>
```

Comment:
This appears to be the Top Nav description.

Please change:

**{{page.title}} is a variation of navigation that is used in lieu of left navigation. It establishes a user's location within an application and shows the path forward to accomplish a goal.**

Reply:
Changes have been done, please review.